### PR TITLE
Feature/railway schema docu

### PR DESCRIPTION
**Pull request by das-g**
### Reviewed by
- [x] @hixi

### CODE REVIEW

**Comment by hixi** [comment]
:+1:

File: docs/schema/osmaxx_schema.yaml [hunk 1]
```diff
@@ -2022,7 +2022,7 @@ layers: !!omap
                 description: 'T-bar lift. Overhead tow-line for skiers and riders with T-shaped carriers for two passengers.'
             - "j-bar":
                 osm_tags:
-                  - "aerialway": 't-bar'
```
Comment (hixi) [inline docs/schema/osmaxx_schema.yaml]: :+1:

**Comment by hixi** [comment]
@das-g: Can this be merged? If not, what is missing?

**Comment by das-g** [comment]
> Can this be merged?

Yes.
> If not, what is missing?

Nothing that I'm aware of. Simply forgot this was still open.

**Pull request merged**